### ISSUE TITLE
Can't zoom using Ctrl+MWheel

### ISSUE THREAD

**Issue by taufiqsun** [atom/atom#13625]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Zoom using `Ctrl-MWheel` does not work in Atom 1.13.0. It worked in Atom 1.12.9.

### Steps to Reproduce

1. Open Atom in safe mode using `atom --safe`
2. Open the `settings-view` and ensure you have the `Zoom Font When Ctrl Scrolling` setting enabled.
3. Try to zoom using `Ctrl-MWheel`.

**Expected behavior:** Font size to change depending on scroll direction.

**Actual behavior:** The Text editor scrolls.

**Reproduces how often:** 100%

#### atom --version
```
Atom    : 1.13.0
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

#### apm --version
```
apm  1.14.1
npm  3.10.5
node 4.4.5
python 2.7.11
git 2.11.0.windows.1
visual studio 2015
```

### Additional Information

`event.target` in [this check](https://github.com/atom/atom/blob/c6db4df6c09d3863043c1ff0e767578db611cdd5/src/workspace-element.coffee#L105) in different in Atom 1.13.0.

### Atom 1.12.9

![1 12 9](https://cloud.githubusercontent.com/assets/1058982/21962470/2b31f4fa-db27-11e6-9932-fedc599ddb78.PNG)

### Atom 1.13.0

![1 13 0](https://cloud.githubusercontent.com/assets/1058982/21962471/2fb09aea-db27-11e6-83a2-d26d43bd26ba.PNG)

----

For some reason I can't scroll zoom in/out in the editor even with the option being checked. Any ideas?

**Comment by KevinNadaud** [comment]
I have this problem to since the upgrade from 1.12.9 to 1.13.0. 
What is your OS ?
I'm using linux mint 18.

**Comment by taufiqsun** [comment]
I'm using windows 10

**Comment by KevinNadaud** [comment]
OK. So it seems to be not OS-related. Your have the 1.13.0 version too ?

**Comment by taufiqsun** [comment]
Yep, I'm using 1.13.0 as well.

**Comment by Ben3eeE** [comment]
I can reproduce this in Atom 1.13.0

**Comment by DFelten** [comment]
Same error here with Atom 1.14.0-beta0.

**Comment by Ben3eeE** [comment]
I edited the body of the issue to include the issue template and steps to reproduce. @taufiqsun in the future please completely fill out the provided issue template.

**Comment by jsolisu** [comment]
**event.target** is needed?

**Comment by ungb** [comment]
I am able to repro this issue as well.  Confirmed worked on 1.12.9 but broke in 1.13.0

**Comment by Zangoran** [comment]
I am able to reproduce this issue as well. I am running 1.14.0-beta1 x64 (Windows). Used to work in 1.13.0 for me.

**Comment by as-cii** [comment]
Thanks for the reports and apologies for this regression. The fix has been merged into master and it will land in Atom 1.13.1 and 1.14.0-beta3.
